lib/fake_braintree/payment_method.rb

```diff
@@ -0,0 +1,11 @@
+require 'active_support/core_ext/hash/keys'
+
+module FakeBraintree
+  class PaymentMethod
+    def self.tokenize_card(attributes)
```

Comment:
It looks like this is only used in tests - does it make more sense to make this a method somewhere in `spec/support`? Is there a reason to add it to the public `FakeBraintree` API that I'm maybe missing?